js/search/search.js

```diff
@@ -113,17 +116,10 @@ class Search {
                 return;
             }
 
-            if (!fs.existsSync(searchConfig.FOLDERS_CONSTANTS.INDEX_PATH)) {
-                log.send(logLevels.ERROR, 'User index folder not found');
-                reject(new Error('User index folder not found'));
-                return;
-            }
-
-            const indexId = randomString();
-            libSymphonySearch.symSECreatePartialIndexAsync(searchConfig.FOLDERS_CONSTANTS.TEMP_BATCH_INDEX_FOLDER, indexId, messages, (err, res) => {
+            libSymphonySearch.symSEIndexMainRAMAsync(messages, function (err, res) {
                 if (err) {
-                    log.send(logLevels.ERROR, 'Batch Indexing: error ->' + err);
-                    reject(new Error(err));
+                    log.send(logLevels.ERROR, `IndexBatch: Error indexing messages to memory : ${err}`);
```

Comment:
have we made sure this doesn't return any user path?

Reply:
This will be part of the `R53` release